pubspec.yaml

```diff
@@ -72,7 +72,10 @@ dependencies:
 
 dependency_overrides:
   async: 2.8.0
-  web3dart: ^2.4.1
```

Comment:
What's the reason this was being replaced with rolly's local web3 repo? Seems a bit odd

Reply:
yeah there is a bug in the web3dart library that he had to override with his own repo... this was the same situation with andrei and firebase... we can move that fix into the AX repo